app/models/book_author.rb

```diff
@@ -0,0 +1,4 @@
+class BookAuthor < ActiveRecord::Base
```

Comment:
Thanks @dstrunk, I probably should have run this by you guys first.  I was thinking about the case of co-authors: https://www.goodreads.com/book/show/67035.Relic?from_search=true&search_version=service  ( Douglas Preston (Goodreads Author), Lincoln Child) - what do you think?